pkg/pipeline/controller.go

```diff
@@ -400,18 +403,14 @@ func (c *Controller) SendEOS(ctx context.Context) {
 			c.p.Stop()
 
 		case livekit.EgressStatus_EGRESS_ACTIVE:
-			c.Info.UpdatedAt = time.Now().UnixNano()
-			if c.Info.Error != "" {
-				c.Info.Status = livekit.EgressStatus_EGRESS_FAILED
-				c.p.Stop()
```

Comment:
so we no longer log it as failed? or stop the process?

Reply:
L401-403:
```
		case livekit.EgressStatus_EGRESS_ABORTED,
			livekit.EgressStatus_EGRESS_FAILED:
			c.p.Stop()
```

The difference is just that we now mark it as failed when setting the error instead of relying on whether or not `Error` is empty